lib/PHPCfg/Parser.php

```diff
@@ -1246,7 +1252,7 @@ private function parseParameterList(Func $func, array $params) {
                 $defaultBlock = null;
             }
             $result[] = $p = new Op\Expr\Param(
-                $this->parseExprNode($param->name),
+                $this->parseExprNode($param->var)->name,
```

Comment:
Shouldn't this be `$this->parseExprNode($param->var->name)`?

Reply:
Both variants work without breaking the tests, but your code should be more appropriate. I'm force pushing it along with removal of condition commented above.